cmd/dep/ensure.go

```diff
@@ -172,8 +172,14 @@ func (cmd *ensureCommand) Run(ctx *dep.Ctx, args []string) error {
 		return errors.Wrap(err, "ensure ListPackage for project")
 	}
 
-	if err := checkErrors(params.RootPackageTree.Packages); err != nil {
+	if fatal, err := checkErrors(params.RootPackageTree.Packages); err != nil && fatal {
 		return err
+	} else if ctx.Verbose {
+		if errs, ok := err.(*pkgtreeErrs); ok {
+			ctx.Out.Println(errs)
+		} else {
+			ctx.Out.Println(err)
+		}
```

Comment:
Do these two behave differently?

Zooming out a bit and considering both: is this equivalent?
```go
if fatal, err := checkErrors(params.RootPackageTree.Packages); err != nil {
  if fatal {
    return err
  } else if ctx.Verbose {
    ctx.Out.Println(err)
  }
}
```